grc/gui_qt/components/dialogs.py

```diff
@@ -170,10 +181,30 @@ def open_chooser(widget=None):
         self.buttonBox.rejected.connect(self.reject)
         self.layout = QVBoxLayout()
         self.layout.addWidget(self.tabs)
+        error_msg.setText('\n\n'.join(self._block.get_error_messages()))
+        self.layout.addWidget(error_msg)
         self.layout.addWidget(self.buttonBox)
 
         self.setLayout(self.layout)
 
+    def find_param_widget(self,button):
+        # Find the correct layout
+        layout = self.tabs.currentWidget().layout()
+        # Find the location in the layout of the clicked button
+        location = layout.getItemPosition(layout.indexOf(button))
+        # The required widget is at column 1
+        return layout.itemAtPosition(location[0], 1).widget()
+
+    def open_filero(self):
+        f_name, fltr = QFileDialog.getOpenFileName(self,"Open a Data File")
```

Comment:
`, `

Reply:
Thanks for your hints. Sorry I was blind. But strange, before uploading I checked with autopep8.
But autopep proposed to insert space between " and the O. Strange.
I'll go to fix it.